examples/flow_cp.cpp

```diff
@@ -358,7 +372,8 @@ try
         }
     } else {
         outputWriter.writeInit( simtimer );
-        std::cout << "\n\n================ Simulation turned off ===============\n" << std::flush;
+        if ( output_cout )
```

Comment:
Nitpick: please add braces. I do notice that you usually use the braces-on-separate-lines style. I usually prefer the slightly more compact initial-brace-on-same-line style for in-function loops and conditionals (for/if/while etc.) but will accept both.